src/config/settings.js

```diff
@@ -39,6 +39,9 @@ export const ADAPT_HEIGHT_TIMEOUT = 50;
 // special id for newly created comments
 export const NEW_COMMENT_ID = '';
 
+// default number of quick replies to show
+export const MAX_QUICK_REPLIES_TO_SHOW = 1;
```

Comment:
This can be changed to a bigger value, i.e. 2 or 3.

Reply:
Agreed. We should probably add overflow x scroll to the div and then you can scroll sideways to find the reply if needed.

Reply:
Maybe default should be 3.